cmd/cli/root.go

```diff
@@ -62,12 +62,16 @@ func init() {
 	cobra.CheckErr(rootCmd.MarkPersistentFlagFilename("key-file", "pem"))
 	cobra.CheckErr(viper.BindPFlag("key_file", rootCmd.PersistentFlags().Lookup("key-file")))
 
+	rootCmd.PersistentFlags().String("manage-api", "127.0.0.1:8888", "Management API address")
+	cobra.CheckErr(viper.BindPFlag("manage_api", rootCmd.PersistentFlags().Lookup("manage-api")))
+
```

Comment:
Per https://github.com/storacha/piri/pull/173/files#r2257733914 we can remove this.